app/components/activities/groups/base_activity_component.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
```

Comment:
Since this base activity component was added here, can you update the other components within the `groups` directory to inherit from `BaseActivityComponent` and remove the `initialize` method from these components as well